scripts/install-gnb.sh

```diff
@@ -29,23 +31,11 @@ Group=nogroup
 [Install]
 WantedBy=multi-user.target
 EOF
-
-# Install build tools
+# \
```

Comment:
Why the `\`?

Reply:
It looked better that way in CodeChat. I can remove it.